lib/matplotlib/tests/test_backend_macosx.py

```diff
@@ -18,3 +22,32 @@ def test_cached_renderer():
     fig = plt.figure(2)
     fig.draw_without_rendering()
     assert fig._cachedRenderer is not None
+
+
+@pytest.mark.backend('macosx')
+def test_savefig_rcparam():
+    # Store the save file so we can overwrite it and
+    # then put back the real one after this
+    old_save_file_func = _macosx.choose_save_file
+
```

Comment:
You can use the `monkeypatch` fixture for this.

Reply:
Good call, I didn't know about that fixture, thanks!